content/200-concepts/100-components/03-prisma-migrate/050-prisma-migrate-flows.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: 'Migration flows'
+metaTitle: 'Migration flows (Concepts)'
+metaDescription: 'Prisma Migrate is a declarative data modeling and schema migration tool that is available via the Prisma CLI.'
```

Comment:
Same comment as before: calling migrate declarative is not strictly accurate and could narrow the perceived value I believe.
I've asked internally as I may be the only one with this concern :)